util/docker/docker.go

```diff
@@ -0,0 +1,67 @@
+package docker
```

Comment:
This code is meant to be reused (hence in a `util` package).  I am planning to change the example swarm flavor to use this instead of including a client in its own `example` package.